app/deployments/routes.py

```diff
@@ -560,7 +585,9 @@ def depaction(depid):
             )
             flash("Action successfully triggered.", "success")
         except Exception as e:
-            app.logger.error("Action on deployment {} failed: {}".format(dep.uuid, str(e)))
+            app.logger.error(
```

Comment:
Again, sometimes we use `format()` and sometimes we use `f-strings`. Be consistent (I would prefer `f-strings` when possibile)